packages/taquito-http-utils/src/fetch-adapter.ts

```diff
@@ -0,0 +1,241 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
```

Comment:
small nitpick, can we ctrl + shift + I this doc. some spacing seems to look a bit weird

Reply:
Done